yarn/src/main/scala/org/apache/spark/deploy/yarn/ApplicationMaster.scala

```diff
@@ -691,11 +691,11 @@ private[spark] class ApplicationMaster(
     }
 
     override def receiveAndReply(context: RpcCallContext): PartialFunction[Any, Unit] = {
-      case RequestExecutors(requestedTotal, localityAwareTasks, hostToLocalTaskCount) =>
+      case r: RequestExecutors =>
```

Comment:
suggestion from an earlier review -- I personally don't have a strong preference, but I agree with the idea that there really isn't anything gained by pulling apart the fields, just to repeat them all again.  Easier to avoid that here and just do `r.foo` in the next few lines.  I don't think there is any hard stance on style for this.  (Lots of examples of this version in sql, though sometimes it seems to have its own style ...)